tests/malloc_thread_safety/Makefile

```diff
@@ -1,9 +1,15 @@
 include ../Makefile.tests_common
 
+PICOLIBC ?= 0
+ifneq (0,$(PICOLIBC))
+  USEMODULE += picolibc
+  FEATURES_REQUIRED += picolibc
```

Comment:
Isn't only `FEATURES_REQUIRED += picolibc` needed?

Reply:
Ah, yes. Since modeling both c libs as features, this workaround is no longer needed, right?

Reply:
And also this makes compilation currently fail in some case because both modules get selected.